README.md

```diff
@@ -1,3 +1,329 @@
+library from C++ in a cross-platform manner. As of this writing the bindings
+are functional with GCC on Linux and Mac OS X systems, as well as with Visual
+C++ 2005 and Visual C++ 2008 on Windows.
+
+**Note that the JACE C++ bindings require Java 7 to build and run. They do
+not work with Java 8.**
+
+###Compile-time dependencies
+
+To build the Bio-Formats C++ bindings from source, the following modules are
+required:
+
+- [Apache Maven](http://maven.apache.org/)
+     
+     Maven is a software project management and comprehension tool. Along with
+     Ant, it is one of the supported build systems for the Bio-Formats Java
+     library, and is used to generate the Bio-Formats C++ bindings.
+
+- [CMake](http://www.cmake.org/)
+     
+     CMake is a cross-platform, open source build system generator, commonly
+     used to build C++ projects in a platform-independent manner. CMake
+     supports GNU make as well as Microsoft Visual Studio, allowing the
+     Bio-Formats C++ bindings to be compiled on Windows, Mac OS X, Linux and
+     potentially other platforms.
+
+- [Boost Thread](http://www.boost.org/)
+     
+     Boost is a project providing open source portable C++ source libraries.
+     It has become a suite of de facto standard libraries for C++. The
+     Bio-Formats C++ bindings require the Boost Thread module in order to
+     handle C++ threads in a platform independent way.
+
+- [Java Development Kit](http://www.oracle.com/technetwork/java/javase/downloads/)
+     
+     Version 6 or 7 is required; version 8 is not currently supported.
+     At runtime, only the Java Runtime Environment (JRE) is necessary
+     to execute the Bio-Formats code. However, the full J2SE
+     development kit is required at compile time on some platforms
+     (Windows in particular), since it comes bundled with the JVM
+     shared library (jvm.lib) necessary to link with Java.
+
+For information on installing these dependencies, refer to the section for
+your specific platform below.
+
+###How to build
+
+The process of building the Bio-Formats C++ bindings is divided into two
+steps:
+
+1. Generate a C++ project consisting of "proxies" which wrap the Java code.
+   This step utilizes the Maven project management tool, specifically a
+   Maven plugin called cppwrap.
+
+2. Compile this generated C++ project. This step utilizes the cross-platform
+   CMake build system.
+
+For details on executing these build steps, refer to the section for your
+specific platform below.
+
+###Build results
+
+If all goes well, the build system will:
+
+1. Generate the Bio-Formats C++ proxy classes
+2. Build the Jace C++ library
+3. Build the Java Tools C++ library
+4. Build the Bio-Formats C++ shared library
+5. Build the showinf and minimum_writer command line tools for testing the
+   functionality.
+
+Please be patient, as the build may require several minutes to complete.
+
+Afterwards, the dist/formats-bsd subdirectory will contain the following
```

Comment:
```
[sbesson@ome-ci-c6-05 ~]$ ls -alh bio-formats-jace/target/cppwrap/build/dist/bio-formats-jace/jar/
total 14M
drwxr-xr-x 2 sbesson lsd  4.0K May 16 09:27 .
drwxr-xr-x 4 sbesson lsd  4.0K May 16 09:27 ..
-rw-r--r-- 1 sbesson lsd   11K May 16 09:27 bio-formats-jace-5.1.10.jar
-rw-r--r-- 1 sbesson lsd  143K May 16 09:27 formats-api-5.1.10.jar
-rw-r--r-- 1 sbesson lsd  676K May 16 09:27 formats-bsd-5.1.10.jar
-rw-r--r-- 1 sbesson lsd  116K May 16 09:27 formats-common-5.1.10.jar
-rw-r--r-- 1 sbesson lsd  2.2M May 16 09:27 guava-17.0.jar
-rw-r--r-- 1 sbesson lsd  1.2K May 16 09:27 jace-runtime.jar
-rw-r--r-- 1 sbesson lsd 1009K May 16 09:27 jai_imageio-5.1.10.jar
-rw-r--r-- 1 sbesson lsd   34K May 16 09:27 jgoodies-common-1.7.0.jar
-rw-r--r-- 1 sbesson lsd  115K May 16 09:27 jgoodies-forms-1.7.2.jar
-rw-r--r-- 1 sbesson lsd  561K May 16 09:27 joda-time-2.2.jar
-rw-r--r-- 1 sbesson lsd  332K May 16 09:27 kryo-2.24.0.jar
-rw-r--r-- 1 sbesson lsd  265K May 16 09:27 logback-classic-1.1.1.jar
-rw-r--r-- 1 sbesson lsd  418K May 16 09:27 logback-core-1.1.1.jar
-rw-r--r-- 1 sbesson lsd  207K May 16 09:27 metadata-extractor-2.6.2.jar
-rw-r--r-- 1 sbesson lsd  4.9K May 16 09:27 minlog-1.2.jar
-rw-r--r-- 1 sbesson lsd   17K May 16 09:27 native-lib-loader-2.0.2.jar
-rw-r--r-- 1 sbesson lsd   41K May 16 09:27 objenesis-2.1.jar
-rw-r--r-- 1 sbesson lsd  594K May 16 09:27 ome-xml-5.1.10.jar
-rw-r--r-- 1 sbesson lsd  108K May 16 09:27 perf4j-0.9.13.jar
-rw-r--r-- 1 sbesson lsd  272K May 16 09:27 serializer-2.7.1.jar
-rw-r--r-- 1 sbesson lsd   29K May 16 09:27 slf4j-api-1.7.6.jar
-rw-r--r-- 1 sbesson lsd  601K May 16 09:27 specification-5.1.10.jar
-rw-r--r-- 1 sbesson lsd  1.3M May 16 09:27 turbojpeg-5.1.10.jar
-rw-r--r-- 1 sbesson lsd  3.1M May 16 09:27 xalan-2.7.1.jar
-rw-r--r-- 1 sbesson lsd  1.2M May 16 09:27 xercesImpl-2.8.1.jar
-rw-r--r-- 1 sbesson lsd  190K May 16 09:27 xml-apis-1.3.04.jar
-rw-r--r-- 1 sbesson lsd  115K May 16 09:27 xmpcore-5.1.2.jar
```

So `jace-runtime.jar` is there as well as the non-uber jars (no `bioformats_package.jar`) by default.